pkgs/development/libraries/rapidjson/default.nix

```diff
@@ -11,6 +11,13 @@ stdenv.mkDerivation rec {
     sha256 = "1jixgb8w97l9gdh3inihz7avz7i770gy2j2irvvlyrq3wi41f5ab";
   };
 
+  patches = [
+    (builtins.fetchurl {
```

Comment:
Please use `fetchpatch`, this will fix the CI failing.

Reply:
re: fetchpatch: was not aware it exists, will do, thanks!
re: comment: what about "make compatible with C++20" is unclear?

Reply:
> re: fetchpatch: was not aware it exists, will do, thanks!

:+1: 

> re: comment: what about "make compatible with C++20" is unclear?

That sound totally clear to me, it would just be nice to have a comment in-code explaining this as well.